scripts/generate/jobs.mjs

```diff
@@ -0,0 +1,456 @@
+                },
+                homepage: 'https://josh.evie.dev',
+                publishConfig: {
+                  access: 'public'
+                }
+              }
+            : {
+                name: `@joshdb/${name}`,
+                version: '1.0.0',
+                description,
+                author: 'Évelyne Lachance <eslachance@gmail.com> (https://evie.codes/)',
+                contributors: [],
+                license: 'Apache-2.0',
+                main: 'dist/index.js',
+                module: 'dist/index.mjs',
+                types: 'dist/index.d.ts',
+                exports: {
+                  import: './dist/index.mjs',
+                  require: './dist/index.js',
+                  types: './dist/index.d.ts'
+                },
+                sideEffects: false,
+                scripts: {
+                  test: 'jest',
+                  lint: 'eslint src tests --ext ts --fix -c ../../.eslintrc',
+                  build: 'rollup -c rollup.config.ts',
+                  prepack: 'rollup-type-bundler',
+                  bump: 'cliff-jumper',
+                  'check-update': 'cliff-jumper --dry-run'
+                },
+                dependencies: {},
+                devDependencies: {
+                  '@favware/rollup-type-bundler': '^1.0.7',
+                  jest: '^27.5.1',
+                  rollup: '^2.70.2',
+                  'standard-version': '^9.3.2'
+                },
+                repository: {
+                  type: 'git',
+                  url: 'git+https://github.com/josh-development/utilities.git'
+                },
+                files: ['dist', '!dist/*tsbuildinfo'],
+                engines: {
+                  node: '>=16.6.0',
+                  npm: '>=7.0.0'
+                },
+                keywords: [],
+                bugs: {
+                  url: 'https://github.com/josh-development/utilities/issues'
+                },
+                homepage: 'https://josh.evie.dev',
+                publishConfig: {
+                  access: 'public'
+                }
+              },
+          null,
+          2
+        )
+      );
+
+      await writeFile(
+        resolvePath(name, 'README.md'),
+        `<div align="center">
+
+![Josh Logo](https://evie.codes/josh-light.png)
+
+# @joshdb/${name}
+
+**${description}**
+
+[![GitHub](https://img.shields.io/github/license/josh-development/utilities)](https://github.com/josh-development/utilities/blob/main/LICENSE)
+[![npm](https://img.shields.io/npm/v/@joshdb/json?color=crimson&logo=npm&style=flat-square&label=@joshdb/${name})](https://www.npmjs.com/package/@joshdb/${name})
+
+</div>
+
+## Description
+
+${description}
+
+## Features
+
+- Written in TypeScript
+- Offers CommonJS and ESM bundles
+- Fully tested
+
+## Installation
+
+You can use the following command to install this package, or replace \`npm install\` with your package manager of choice.
+
+\`\`\`sh
+npm install @joshdb/${name}
```

Comment:
Can we add yarn too, pretty please?

Reply:
I'm removing the Yarn part because someone later on will ask, "can we add pnpm please"